idseq_dag/steps/run_alignment_remotely.py

```diff
@@ -404,16 +422,24 @@ def run_chunk(self, part_suffix, input_files, chunk_count, lazy_run):
             if status == "SUCCEEDED":
                 break
             if status == "FAILED":
-                log.log_event("alignment_batch_error", values={"job_id": job_id})
+                log.log_event("alignment_batch_job_failed", values={'job_id': job_id, 'chunk_id': chunk_id, 'alignment_algorithm': self.alignment_algorithm})
```

Comment:
nit: Is it not better keep it as errors and then specify details on the values? If we want to count success vs. errors might be easier.